<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseForm</class>
 <widget class="QMainWindow" name="DatabaseForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>399</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sahel</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="accessibleName">
         <string>class2</string>
        </property>
        <property name="title">
         <string>  مدیریت پایگاه داده  </string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QCheckBox" name="selectAllCheckBox">
               <property name="accessibleName">
                <string>class2</string>
               </property>
               <property name="text">
                <string>انتخاب همه   </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="selectNoneCheckBox">
               <property name="accessibleName">
                <string>class2</string>
               </property>
               <property name="text">
                <string>هیچکدام   </string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListWidget" name="listWidget">
             <property name="font">
              <font>
               <family>Sahel</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="accessibleName">
              <string>class2</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="labelTotal">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="accessibleName">
                <string>classFDs</string>
               </property>
               <property name="text">
                <string>تعداد کل:</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>10</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="labelSelectedCount">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="accessibleName">
                <string>classFDs</string>
               </property>
               <property name="text">
                <string>انتخاب‌شده:</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="btnAddPoet">
               <property name="accessibleName">
                <string>class2</string>
               </property>
               <property name="text">
                <string>اضافه</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnRemovePoet">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="accessibleName">
                <string>class2</string>
               </property>
               <property name="text">
                <string>حذف</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnExport">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="accessibleName">
                <string>class2</string>
               </property>
               <property name="text">
                <string>خروجی</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnDownloadForm">
               <property name="accessibleName">
                <string>class2</string>
               </property>
               <property name="text">
                <string>  دانلود از مخزن  </string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnCompactDB">
               <property name="accessibleName">
                <string>class2</string>
               </property>
               <property name="text">
                <string>  فشرده‌سازی پایگاه داده  </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelProgress">
        <property name="accessibleName">
         <string>class2</string>
        </property>
        <property name="text">
         <string>لطفا منتظر بمانید…</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="accessibleName">
         <string>class2</string>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label">
          <property name="accessibleName">
           <string>class2</string>
          </property>
          <property name="text">
           <string>مسیر پایگاه داده:  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="accessibleName">
           <string>class2</string>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(21, 21, 21);</string>
          </property>
          <property name="text">
           <string>databasePath</string>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClose">
          <property name="accessibleName">
           <string>class2</string>
          </property>
          <property name="text">
           <string>بستن</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
